browser/app/js/components/Login.js

```diff
@@ -62,6 +62,17 @@ export default class Login extends React.Component {
       type: 'danger',
       message: ''
     }))
+
+    // If we're on an insecure connection, warn the user.
+    if (window.location.protocol === 'http:') {
+      dispatch(actions.showAlert({
+        type: 'danger',
+        message: 'You are on an insecure connection. Your access key and secret '
+               + 'key should be considered exposed to the network upon login. '
+               + 'Please configure certificates for HTTPS access.'
```

Comment:
Done! Looks like this:
![screen shot 2017-07-24 at 11 11 45 am](https://user-images.githubusercontent.com/3038254/28537825-0b1f0b9c-7061-11e7-98f1-be7fdc8e6636.png)